README.md

```diff
@@ -92,9 +99,16 @@ or
   git push -u origin main
 ```
 
-- Navigate to your fork, on the top of the files section you'll notice a new section containing, a contribute button!
-- Click on the contribute button, it will open a drop down, click the pull request button on the drop down.
-  Note: A pull request allows your changes to be merged with the original project.
+In case you get an error message like the one below, its likely you forgot to fork the repo before cloning it. To fix this, its best to start over with the How to Contribute section above, and fork the project repo first.
+```
+ERROR: Permission to EddieHubCommunity/hacktoberfest-practice.git denied to <your-github-username>.
+fatal: Could not read from remote repository.
+Please make sure you have the correct access rights and the repository exists.
+```
+
+- On the GitHub website, navigate to your forked repo - on the top of the files section you'll notice a new section containing a `Compare & Pull Request` button!
+- Click on that button and this will load a new page, comparing the local branch in your forked repo, against the main branch in the EddieHub Hacktoberfest repo. Accept the default values in the drop down boxes and click the green `Create Pull Request` button. The next page will run a check and a bot should give you a success message.  
```

Comment:
I would change the sentence 'the next page will run a check'. It is not the page that is running the check. Maybe remove that sentence or write something like: after creating the pr our github action workflow will add a welcome message to your pr.

Reply:
no worries; updated wording per your suggestion